state/state.go

```diff
@@ -1844,6 +1844,22 @@ func (s *State) GetNetworkBootStrapIdentity() interfaces.IHash {
 	return primitives.NewZeroHash()
 }
 
+// The identity for validating messages
+func (s *State) GetNetworkSkeletonIdentity() interfaces.IHash {
+	switch s.NetworkNumber {
+	case constants.NETWORK_MAIN:
+		return primitives.NewZeroHash()
+	case constants.NETWORK_TEST:
+		return primitives.NewZeroHash()
+	case constants.NETWORK_LOCAL:
+		id, _ := primitives.HexToHash("88888847f6cd639255df8f6f9e4f015058c93bc02e72f8e1287d7ff0d3fc184b")
```

Comment:
Define as a CONST elsewhere.  
  * Improves readability
  * decreases likelihood of cut/paste
  * limits scope of changes in case of repetitions (e.g. `GetNetworkBootStrapIdentity` above)